sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -353,35 +353,72 @@ def test_gscv_backends(backend_set):
     gscv.fit(y, X)
 
 
+def test_BaseGridSearch(
```

Comment:
I don't know if it's just me, but this seems a bit strange way to set up the different instances for testing. Won't setting up 3 fixtures, and choose between them, or choose a common parameter grid and passing that by position and then initiate selected class instance be simpler? 

(This is completely subjective, so non-blocking)

Reply:
You are right. I was trying not to put anything in the gloabal namespace just in case any other module imported this one, but I ended up overengineering it.